core/field_dropdown.ts

```diff
@@ -217,16 +217,16 @@ export class FieldDropdown extends Field {
   protected shouldAddBorderRect_(): boolean {
     return !this.getConstants()!.FIELD_DROPDOWN_NO_BORDER_RECT_SHADOW ||
         this.getConstants()!.FIELD_DROPDOWN_NO_BORDER_RECT_SHADOW &&
-        !this.getSourceBlock().isShadow();
+        !this.getSourceBlock()?.isShadow();
   }
 
   /** Create a tspan based arrow. */
   protected createTextArrow_() {
     this.arrow_ = dom.createSvgElement(Svg.TSPAN, {}, this.textElement_);
     this.arrow_!.appendChild(document.createTextNode(
-        this.getSourceBlock().RTL ? FieldDropdown.ARROW_CHAR + ' ' :
-                                    ' ' + FieldDropdown.ARROW_CHAR));
-    if (this.getSourceBlock().RTL) {
+        this.getSourceBlock()?.RTL ? FieldDropdown.ARROW_CHAR + ' ' :
+                                     ' ' + FieldDropdown.ARROW_CHAR));
```

Comment:
ew clang why are we doing vertical alignment 

(no action needed just, ew clang)